### PR TITLE
[EasyActivity] Add simple documentation

### PR DESCRIPTION
**Pull request by mvlad**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes

### CODE REVIEW

File: packages/EasyActivity/docs/activitiy-log-entry.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+The **ActivityLogEntry** class defines an activity log data.
+
+## Properties
+
+A `ActivityLogEntry` object has the following properties:
+
+- `action`: an action type, one of: `ActivityLogEntry::ACTION_CREATE`, `ActivityLogEntry::ACTION_DELETE`, or `ActivityLogEntry::ACTION_UPDATE`.
+- `actorId`: an optional "Actor" identifier in an application.
+- `actorName`: an optional "Actor" name in an application.
+- `actorType`: a mandatory "Actor" type, can be a "user", "provider", "customer", "jwt:provider", "api_key:customer", or something similar in an application. The default value is `ActivityLogEntry::DEFAULT_ACTOR_TYPE` (`system`).
```
[the remark this answers is not in the document]
Comment (roman-eonx) [reply packages/EasyActivity/docs/activitiy-log-entry.md:17]: I think these examples will guide the apps to use similar values and not invent their own. The "or something similar" phrase doesn't conflict with "whatever" :wink:

File: packages/EasyActivity/docs/usage.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+---eonx_docs---
+title: Usage
+weight: 1002
+---eonx_docs---
+
+# Usage
+
+A [log DB entry](1) is stored in the `easy_activity_logs` table by default (the table name can be changed in the config). 
+An application can either use `DoctrineDbalStatementsProvider` to create a table or describe a DB entity/model relying on this table by itself.
```
[the remark this answers is not in the document]
Comment (roman-eonx) [reply packages/EasyActivity/docs/usage.md:9]: Agree with FQCN.

File: packages/EasyActivity/docs/symfony/config.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+#            nested_object_allowed_properties:
+#                App\Entity\SomeAnotherEntity:
+#                    - processingDate
+#
+```
+
+#### Settings for entities
+* `disallowed_properties` — an optional global array of "Subject" property names to be excluded from log entries (this list will be applied to all "Subjects").
+* `subjects` — a list of "Subject" classes to be logged. Each item can contain the following params:
+  * `allowed_properties` — an optional array of "Subject" property names to be "whitelisted" for log entries.
```
[the remark this answers is not in the document]
Comment (nick-lc) [reply packages/EasyActivity/docs/symfony/config.md:34]: Hey @roman-eonx , what is the default behaviour, in terms of what is included in log entries?

I can see how `disallowed_properties` will exclude properties from the log entry, but what does `allowed_properties` do? If it exists, does it restrict what will be included?

[the remark this answers is not in the document]
Comment (roman-eonx) [reply packages/EasyActivity/docs/symfony/config.md:34]: Hey @nick-lc,
Yes, `allowed_properties` will restrict the list of properties (whitelist them). The default behavior is to pass all the properties to logs.

File: packages/EasyActivity/docs/symfony/config.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  * `disallowed_properties` — an optional array of "Subject" property names to be excluded from log entries.
+  * `nested_object_allowed_properties` — an optional property names to be "whitelisted" for log related entries.
+  * `type` — an optional "Subject" type mapping. If no type is provided, a FQCN will be used by default.
```
[the remark this answers is not in the document]
Comment (mvlad) [reply packages/EasyActivity/docs/symfony/config.md:38]: Added the `type` to the config example.

File: packages/EasyActivity/docs/symfony/config.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (mvlad) [reply packages/EasyActivity/docs/symfony/config.md]: Added the information about default values of `allowed_properties` and `disallowed_properties` settings.
Please, let me know if we need to describe anything else.

File: packages/EasyActivity/docs/usage.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (mvlad) [reply packages/EasyActivity/docs/usage.md]: Changed class names to FQCN.

**Comment by nick-lc** [comment]
Hi guys, I made some changes to the docs. Hopefully I haven't changed the meaning of anything!

**Review by roman-eonx — DISMISSED** [review]

File: packages/EasyActivity/docs/usage.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+- `EonX\EasyActivity\Logger\SyncActivityLogger` to save an activity log entry synchronously
+
+## Symfony bridge
+
+The Symfony bridge provided with this package allows it to be integrated into a Symfony-based application. Besides
+Symfony bundle/extension classes, it brings the following functionality:
+
+- The default implementation for `EonX\EasyActivity\Interfaces\ActivitySubjectDataSerializerInterface`:
+  `EonX\EasyActivity\Bridge\Symfony\Serializers\SymfonyActivitySubjectDataSerializer`, which is a simple wrapper for
+  `\Symfony\Component\Serializer\SerializerInterface` used to serialize activity log entry data. Please note that all
```
Comment (voodooism) [inline packages/EasyActivity/docs/usage.md:62]: I see that we don't start FQCN from the slash in most cases. Delete it pls.

**Review by voodooism — CHANGES_REQUESTED** [review]

File: packages/EasyActivity/docs/activity-log-entry.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+## Properties
+
+An ActivityLogEntry object has the following properties:
+
+- `action`: The type of action performed on the subject, which may be one of:
+  - `ActivityLogEntry::ACTION_CREATE` (i.e. `create`): Create subject
+  - `ActivityLogEntry::ACTION_DELETE` (i.e. `delete`): Delete subject
+  - `ActivityLogEntry::ACTION_UPDATE` (i.e. `update`): Update subject
+- `actorId`: An optional identifier for an actor in the application.
+- `actorName`: An optional nam for an actor in the application.
```
[the remark this answers is not in the document]
Comment (nick-lc) [reply packages/EasyActivity/docs/activity-log-entry.md]: Thx!

File: packages/EasyActivity/docs/usage.md (hunk 5 above)
Comment (nick-lc) [reply packages/EasyActivity/docs/usage.md]: Thx!

**Review by roman-eonx — APPROVED** [review]

**Review by voodooism — APPROVED** [review]

**Review by nick-lc — APPROVED** [review]

**Pull request merged**